recipe/meta.yaml

```diff
@@ -24,78 +24,76 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
+    - cython >=3.0.5
     - numpy {{ numpy }}
     - versioneer
-    - tomli # [py<311]
     - pip
```

Comment:
```suggestion
    - pip
    - wheel
```

Reply:
https://github.com/pandas-dev/pandas/blob/v2.2.1/pyproject.toml#L7

Reply:
Done!

Reply:
setuptools should not be needed and wheel is also not needed. We use meson+meson-python to build the project, not setuptools.

Reply:
Ah okay, got it! I hadn't realize meson-python was an alternative to setuptools until now. Thanks!